04.class/memo.js

```diff
@@ -0,0 +1,94 @@
+import { MemoManager } from "./src/MemoManager.js";
+import readline from "readline";
+
+const memoManager = new MemoManager();
+
+async function main() {
+  await memoManager.init();
+  await processCommand(process.argv.slice(2));
+}
+
+async function processCommand(args) {
+  switch (args[0]) {
+    case undefined:
+      await addMemo();
+      break;
+    case "-l":
+      await listMemos();
+      break;
+    case "-r":
+      await displayMemo();
+      break;
+    case "-d":
+      await deleteMemo();
+      break;
+    case "-e":
+      await editMemo();
+      break;
+    default:
+      console.log("無効なオプションです。");
+  }
+}
+
+async function listMemos() {
+  const memoTitles = memoManager.listMemoTitles();
```

Comment:
```suggestion
  const titles = memoManager.listMemoTitles();
```

でもよさそうです
https://bootcamp.fjord.jp/pages/avoid-too-specific-var-name